### PR TITLE
Feature/merkleeyes to iavl

### PR DESCRIPTION
**Pull request by ebuchman**
replaces https://github.com/tendermint/tendermint/pull/737

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -25,12 +25,12 @@ import:
...
   - data
-- package: github.com/tendermint/merkleeyes
-  version: master
+- package: github.com/tendermint/iavl
+  version: develop
   subpackages:
   - iavl
 - package: github.com/tendermint/tmlibs
-  version: ~0.3.2
+  version: develop
```
Comment (zramsay) [inline glide.yaml]: won't these changes make `go get` easily breakable? also contradicts: https://github.com/tendermint/coding/issues/24

Comment (melekes) [reply glide.yaml]: only on develop, which is fine. we will change these back to tags when we'll be doing a release

Comment (zramsay) [reply glide.yaml]: ah gotcha

Comment (ebuchman) [reply glide.yaml]: exactly. we can include that in the guidelines

**Pull request merged**